### ISSUE TITLE
Support Pip version 10.0.0

### ISSUE THREAD

**Issue by amos-o** [biosustain/potion#134]
Flask Potion doesn't currently play nice with pip v10.0.0 because of some breaking changes introduced in pip. Specifically here is a sample of the error I run into. 

```
  File "/Users/amos/Envs/moved-api/lib/python3.6/site-packages/flask_potion/filters.py", line 1, in <module>
    from pip.utils import cached_property
ModuleNotFoundError: No module named 'pip.utils'
```

The `pip.utils` module which Potion depends on seems to be deprecated in pip 10.

**Comment by mynameiswillporter** [comment]
I am also receiving this error.

**Comment by mynameiswillporter** [comment]
For those of you who need a workaround, this still works with version 9.0.3.  To downgrade you can do the following: `pip install pip==9.0.3`

**Comment by lyschoening** [comment]
Sorry for the slow response. This should be fixed in v0.15.0.
